mock/nebula/etc/nebula-storaged.conf

```diff
@@ -15,68 +15,80 @@
 --logbufsecs=0
 
 ########## networking ##########
-# Meta server address
---meta_server_addrs=127.0.0.1:45500
-# Local ip
+# Comma separated Meta server addresses
+--meta_server_addrs=127.0.0.1:9559
+# Local IP used to identify the nebula-storaged process.
+# Change it to an address other than loopback if the service is distributed or
+# will be accessed remotely.
 --local_ip=127.0.0.1
 # Storage daemon listening port
---port=44500
+--port=9779
 # HTTP service ip
---ws_ip=127.0.0.1
+--ws_ip=0.0.0.0
 # HTTP service port
---ws_http_port=12000
+--ws_http_port=19779
 # HTTP2 service port
---ws_h2_port=12002
-# heartbeat with meta service
---heartbeat_interval_secs=10
+--ws_h2_port=19780
 
 ######### Raft #########
 # Raft election timeout
 --raft_heartbeat_interval_secs=30
 # RPC timeout for raft client (ms)
 --raft_rpc_timeout_ms=500
 ## recycle Raft WAL
---wal_ttl=3600
+--wal_ttl=14400
 
 ########## Disk ##########
 # Root data path. Split by comma. e.g. --data_path=/disk1/path1/,/disk2/path2/
 # One path per Rocksdb instance.
 --data_path=data1,data2,data3
 
-############## Rocksdb Options ##############
 # The default reserved bytes for one batch operation
 --rocksdb_batch_size=4096
-
-# The default block cache size used in BlockBasedTable. (MB)
-# recommend: 1/3 of all memory
+# The default block cache size used in BlockBasedTable.
+# The unit is MB.
 --rocksdb_block_cache=4
+# The type of storage engine, `rocksdb', `memory', etc.
+--engine_type=rocksdb
 
 # Compression algorithm, options: no,snappy,lz4,lz4hc,zlib,bzip2,zstd
 # For the sake of binary compatibility, the default value is snappy.
 # Recommend to use:
 #   * lz4 to gain more CPU performance, with the same compression ratio with snappy
 #   * zstd to occupy less disk space
 #   * lz4hc for the read-heavy write-light scenario
---rocksdb_compression=snappy
+--rocksdb_compression=lz4
```

Comment:
For what purpose?

Reply:
This is the newest config file in nebula 2.0.